docs/_documentations/troubleshooting.md

```diff
@@ -292,6 +292,18 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
 
 **Workaround:** Review the preceding list and ensure that your project conforms to all of the items in the list.
 
+<!--
+Action/Topic: Understanding application metrics
```

Comment:
John mentioned that this does not look like the right Action/Topic for the server log level setting.

Reply:
@eharris369 Thanks for reminding me! What about putting it under "Checking the application and build statuses"?

Reply:
Can it just go under its own topic?

Reply:
Yes, since that sounds like it is pretty unique, we can move it to its own section.